index.js

```diff
@@ -334,6 +334,10 @@ module.exports = async function(options) {
       synth.define(name, options);
     });
 
+    // Apostrophe prefers that any improvements to @apostrophecms/global
+    // be applied before any project level version of @apostrophecms/global
+    synth.applyImprovementsBeforeProjectLevel();
```

Comment:
This requires a new pass be made at the end after defining the modules. The cleanest way to do it was as an optional method of moog-require that apostrophe always calls. moog-require is internal now, so this isn't something third parties ever have to think about.